pages/glossary/namespaced-element.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Namespaced Element
+key: namespaced-element
+unambiguous: true
+objective: true
+---
+
+An [element][] with a specific [namespaceURI][] value from [HTML namespaces][]. For example an "SVG element" is an element with the "SVG namespace", which is `http://www.w3.org/2000/svg`.
+
+Namespaced elements are not limited to elements described in a specification. They also include custom elements. Elements such as `a`, `title`, and `svg` have a different namespace depending on where they are used. For example a `title` in an HTML page usually has the HTML namespace. When used in an `svg` element, a `title` elements has the SVG namespace instead.
```

Comment:
I actually think that a `<svg>` element inside an HTML document is in the SVG namespace…

[`svg` element in HTML specs](https://html.spec.whatwg.org/multipage/embedded-content-other.html#svg-0) links to [SVG `foreignObject`](https://svgwg.org/svg2-draft/embedded.html#ForeignObjectElement) which explains it's here for inclusion in other XML docs and has a note saying:
> The HTML parser treats elements inside the ‘`foreignObject`’ equivalent to elements inside an HTML document fragment. Any `svg` or `math` element, and their descendents, will be parsed as being in the SVG or MathML namespace, respectively;

Reply:
Good catch! You are right, `svg` has the SVG namespace in HTML.